internal/pkg/deploy/cloudformation/stack/app.go

```diff
@@ -21,29 +21,44 @@ import (
 // DeployedAppMetadata wraps the Metadata field of a deployed
 // application StackSet.
 type DeployedAppMetadata struct {
-	Metadata AppResourcesConfig `yaml:"Metadata"`
+	Metadata AppResources `yaml:"Metadata"`
 }
 
-// AppResourcesConfig is a configuration for a deployed Application
-// StackSet.
+// AppResources is a configuration for a deployed Application StackSet.
+type AppResources struct {
```

Comment:
do we need this? why not only have `AppResourcesConfig`

Reply:
because otherwise we won't be able to do the custom Unmarshal. 
```golang
func (s *AppResourcesConfig) UnmarshalYAML(value *yaml.Node) error {
	if err := value.Decode(&s); err != nil {
		return err
	}
```
just won't work.

Reply:
ah gotcha 👍